sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2463,22 +2463,22 @@ object functions {
    * Splits str around matches of the given regex.
```

Comment:
nit: This general comment refers to parameters, it seems. And `pattern` was replaced by `regex` here in in the original PR. How about _Splits str around matches of the given **pattern**_